set-up-point-cloud/free-trial-limits.md

```diff
@@ -18,6 +18,10 @@ You'll be able to utilize most of the Syskit Point features while in the free tr
 * **PDF exports will have a watermark** stating you exported the file during the Syskit Point Trial
 * **OneDrive data will not be synced** and will not be available for access to gain more details
 * **You will have a limit** of running 10 File Version Cleanup actions
+* [**The Storage Versioning Limit action**](../storage-management/versioning-limits.md) has the following limitations during trial:
+  * Setting version limits is blocked during trial
+  * The Clean Up Existing File Versions has a limit of 5 actions by default during trial
```

Comment:
ovdje mi je malo čudno nešto. ako je setting version limit akcija blocked, a Existing File Versions Cleanup je ustvari opcija na dijalogu te akcije, onda je tu nešto krivo
moguće da se radi o Clean Up File Versions akciji (bez Existing); malo je puno confusing, slična su imena 😄